unionNew.js

```diff
@@ -0,0 +1,13 @@
+function union() {
+    let set = new Set(arguments[0]);
```

Comment:
hm, arguments, now we have rest operator that can be used in arguments https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Functions/Rest_parameters

Reply:
additionally plz look at spread operator and concat function

Reply:
how can the concat function be used?

Reply:
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/concat

review its arguments + spread operator

Reply:
Thank's) fixed in 5c7dc590